db/migrate/20181215145329_create_users.rb

```diff
@@ -3,7 +3,6 @@ def change
     create_table :users do |t|
       t.string :name
       t.string :email
-      t.string :password
```

Comment:
oh wow, co tu sie zadzialo, tak sie tego nie robi, jesli chcesz usunac pole to dodajesz migracje usuwajaca pole a nie rollback i zmiany w migracji tworzacej, bo reszta developerow dana migracje ma juz puszczona i zeby teraz miec wersje bazy taka jak u Ciebie w PR kazdy musi albo kombinowac jak kon pod gore usuwajac informacje o tej migracji z tabeli trzymajacej migracje albo odbudowywac cala baze.

Reply:
Zostało naprawione poprzez dodanie migracji remove_pasword_from_users